app/views/content/life-as-a-teacher/change-careers/benefits-of-changing-careers/_article.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <p>Teachers come from lots of different backgrounds and have often worked in previous careers before becoming teachers.</p>
```

Comment:
```suggestion
    <p>Teachers come from lots of different backgrounds and have often worked in previous careers before making a change to teaching.</p>
```
Worth tweaking slightly to remove the repetition of `teachers`? (But reword better than I have!)